code/datums/gamemode/factions/malf.dm

```diff
@@ -80,5 +82,5 @@ You should now be able to use your Explode spell to interface with the nuclear f
 
 /datum/faction/malf/get_statpanel_addition()
 	if(stage >= FACTION_ENDGAME)
-		return "Time left: [max(AI_win_timeleft/(apcs/3), 0)]"
+		return "Station integrity: [round(100-(AI_win_timeleft/MALF_INITIAL_TIMER)*100)]%"
```

Comment:
doesn't this show 100% integrity if timeleft = 0, while it shows 0% integrity if timeleft = initial timer?

100-0*100=100
100-1*100=0